CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFileManager+FileManagement.swift

```diff
@@ -141,6 +141,29 @@ extension CEWorkspaceFileManager {
         }
     }
 
+    /// This function deletes multiple files or folders from the current project by erasing immediately.
+    /// - Parameters:
+    ///   - files: The files to delete
+    ///   - confirmDelete: True to present an alert to confirm the delete.
+    public func batchDelete(files: Set<CEWorkspaceFile>, confirmDelete: Bool = true) {
+        let deleteConfirmation = NSAlert()
+        deleteConfirmation.messageText = "Do you want to delete \(files.count) items?"
+        deleteConfirmation.informativeText = "These items will be deleted immediately. You can't undo this action."
```

Comment:
This is what Finder says when you try to delete multiple items immediately (I couldn't figure out how to delete files immediately in Xocde).
```suggestion
        deleteConfirmation.messageText = "Are you sure you want to delete the \(files.count) selected items?"
        deleteConfirmation.informativeText = "\(files.count) items will be deleted immediately. You cannot undo this action."
```